apps/game_backend/priv/repo/migrations/20240725215038_add_name_to_the_map.exs

```diff
@@ -0,0 +1,9 @@
+defmodule GameBackend.Repo.Migrations.AddNameToTheMap do
+  use Ecto.Migration
+
+  def change do
+    alter table(:map_configurations) do
+      add :name, :string
+    end
+  end
```

Comment:
Would be nice to have an `execute/2` here that will set the map name to `Araban` so we don't need to remember to do it ourselves

Reply:
I think I can achieve it by updating all map configurations 😆

Reply:
I think that if you add an execute statement in a change method you can't rollback the migration right? maybe you need to specify an `up` and  `down` method

Reply:
`execute/2` lets you do the rollback (up/down), for this particular case some thing like

```elixir
execute(
  "update map_configurations set name = 'Araban'", 
  ""
)
```

PS: `execute/2` written from thin air so verify if plan to use